tutorials/CONTRIBUTING.md

```diff
@@ -0,0 +1,231 @@
+# Contributing to `dingo`
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to dingo, 
+which are hosted in the [GeomScale Organization](https://github.com/GeomScale) on GitHub. 
+These are mostly guidelines, not rules. 
+Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+## Table of Contents
+
+  * [Prerequisites (how to start)](#prerequisites-how-to-start)
+  * [Testing the development branch of `dingo` (get the tools ready)](#testing-the-development-branch-of-dingo-get-the-tools-ready)
+  * [Fork  `dingo` repository (this is your repo now!)](#fork-dingo-repository-this-is-your-repo-now)
+    + [Verify if your fork works (optional)](#verify-if-your-fork-works-optional)
+  * [Working with `dingo` (get ready to contribute)](#working-with-dingo-get-ready-to-contribute)
+    + [GitFlow workflow](#gitflow-workflow)
+    + [Create new branch for your work](#create-new-branch-for-your-work)
+    + [Verify your new branch (optional)](#verify-your-new-branch-optional)
+  * [Modify the branch (implement, implement, implement)](#modify-the-branch-implement-implement-implement)
+    + [Tests](#tests)
+    + [Push](#push)
+  * [Pull request (the joy of sharing)](#pull-request-the-joy-of-sharing)
+  * [Review (ok this is not an exam)](#review-ok-this-is-not-an-exam)
+  
+## Prerequisites (how to start)
+
+* git (see [Getting Started with Git](https://help.github.com/en/github/using-git/getting-started-with-git-and-github))
+* a compiler to run tests - gcc, clang, etc.
+* configured GitHub account
+ 
+Other helpful links:
+
+* http://git-scm.com/documentation
+* https://help.github.com/articles/set-up-git
+* https://opensource.com/article/18/1/step-step-guide-git
+
+## Testing the development branch of dingo (get the tools ready)
+
+Clone the repository, 
+
+    git clone git@github.com:geomscale/dingo.git dingo
+    cd dingo
+    git branch -vv
+
+the last command should tell you that you are in `develop` branch.
+
+Now you need to get the `volesti` sumbodule that `dingo` makes use of. 
+
+To do so, you need to run 
+
+    git submodule update --init
+
+Now get the `boost` library:
+
+    wget -O boost_1_76_0.tar.bz2 https://boostorg.jfrog.io/artifactory/main/release/1.76.0/source/boost_1_76_0.tar.bz2 
+    tar xjf boost_1_76_0.tar.bz2
+    rm boost_1_76_0.tar.bz2
+
+And now you are ready to compile `dingo`
+
+    python setup.py install --user 
+
+Once the last command is completed, you may check if everythings is fine by running some `dingo` tests
+
+    python tests/unit_tests.py
+
+
+If everything is ok, you will see something like this:
+
+[![asciicast](https://asciinema.org/a/3IwNykajlDGEndX2rUtc0D2Ag.svg)](https://asciinema.org/a/3IwNykajlDGEndX2rUtc0D2Ag)
```

Comment:
this is very cool!!!
is it possible to embed a gif so the reader will not have to click to see the video